resources/js/components/fieldtypes/DateFieldtype.vue

```diff
@@ -7,8 +7,7 @@
     	</button>
 
         <div v-if="hasDate || config.inline"
-            class="date-time-container flex flow-col @sm:flex-row"
-            :class="config.time_seconds_enabled ? 'space-x-1' : 'space-x-3'"
+            class="date-time-container flex flex-col @sm:flex-row gap-2"
```

Comment:
@jasonvarga — looks like `flow-col` was a mistake and caused this view to always have a row-based flex setup (instead of conditionally if the container was at the small size).

While this means the time stacks in a sidebar context, I think this makes this more legible for both the date and the time interfaces.